packages/flutter/test/widgets/media_query_test.dart

```diff
@@ -91,6 +93,7 @@ void main() {
     expect(copied.padding, const EdgeInsets.all(9.10938));
     expect(copied.viewPadding, const EdgeInsets.all(11.24031));
     expect(copied.viewInsets, const EdgeInsets.all(1.67262));
+    expect(copied.systemGestureInsets, const EdgeInsets.all(1.5556));
```

Comment:
Separate PR: define these double values with a consts and explain why we're using strange values.

Reply:
Filed https://github.com/flutter/flutter/issues/39012